Dynatrace/dynatrace-operator
# Limited security context of init container

# Description
With this PR the security context of the init container is limited as much as possible.

## How can this be tested?
Run an application in a labeled namespace, such that the app pod gets injected and check the init containers security context.

## Checklist
